docs/authentication.md

```diff
@@ -110,6 +110,6 @@ django-ansible-base comes with many types of authenticators which can be found i
 
 If you wanted to remove authenticators from your application there are two ways to do this:
 1. Remove the unwanted files from your ansible_base installation.
-2. Create a new class directory in your application and only add in the authenticators you care about and then set `ANSIBLE_BASE_AUTHENTICATOR_CLASS_PREFIX` to be the prefix for your class.
+2. Create a new class directory in your application and only add in the authenticators you care about and then set `ANSIBLE_BASE_AUTHENTICATOR_CLASS_PREFIXS` to be the prefix for your class.
```

Comment:
```suggestion
2. Create a new class directory in your application and only add in the authenticators you care about and then set `ANSIBLE_BASE_AUTHENTICATOR_CLASS_PREFIXES` to be the prefix for your class.
```

Reply:
Thanks for reminding me to add my spell checker pre-commit user hook :)